packages/backend/src/services/SavedChartsService/SavedChartService.ts

```diff
@@ -664,7 +665,8 @@ export class SavedChartService {
                         ? []
                         : scheduler.targets.map(getSchedulerTargetType),
```

Comment:
why is this not failing if `filtersUpdatedNum` is not optional ?

Reply:
here it uses `SchedulerUpsertEvent` which doesn't include `filtersUpdatedNum` 
```ts
export type SchedulerUpsertEvent = BaseTrack & {
    event: 'scheduler.created' | 'scheduler.updated';
    userId: string;
    properties: {
        projectId: string;
        organizationId: string;
        schedulerId: string;
        resourceType: 'dashboard' | 'chart';
        cronExpression: string;
        cronString: string;
        resourceId: string;
        format: SchedulerFormat;
        targets: Array<{
            schedulerTargetId: string;
            type: 'slack' | 'email';
        }>;
    };
};
```

In `DashboardService` it uses `SchedulerDashboardUpsertEvent` type, which includes that property `filtersUpdatedNum` as required 👍